src/account.rs

```diff
@@ -36,12 +38,13 @@ pub async fn account_prompt(account_handle: AccountHandle) -> Result<(), Error>
 }
 
 // loop on the account prompt
-pub async fn account_prompt_internal(account_handle: AccountHandle) -> Result<bool, Error> {
+pub async fn account_prompt_internal(account_handle: AccountHandle, history: &mut AccountHistory) -> Result<bool, Error> {
     let alias = {
         let account = account_handle.read().await;
         account.alias().clone()
     };
-    let command: String = Input::new().with_prompt(format!("Account \"{}\"", alias)).interact()?;
+    
+    let command: String = Input::new().with_prompt(format!("Account \"{}\"", alias)).history_with(history).interact_text()?;
```

Comment:
I feel like I remember we had an issue with `interact_text` in the past. @samuel-rufi I think you had something to do with it ?

Reply:
Oh right, it was changed here https://github.com/iotaledger/cli-wallet/pull/136

Reply:
Was there any reason you changed @hauju ? Or can we revert ?

Reply:
With interact only, it does not work for me because the arrow keys are recognised as input. 
Output:
`Account "hauju": ^[[A^[[A`

Help says:

> Allows any characters as input, including e.g arrow keys. Some of the keys might have undesired behavior. For more limited version, see interact_text.